diagnostic.md

```diff
@@ -20,14 +20,19 @@ machine serving the donuts is `http://www.example.com`.
 Write the `curl` request you'd use to retrieve a list of all donuts on the
 server.
 
-```sh
-# your answer here
-```
+curl --include --request "GET" "http://www.example.com"
 
 Write an AJAX request to retrieve a list of all donuts on the server.
 
 ```js
-let getDonuts = /* your answer here */;
+let getDonuts = function (data); {
+  return $.ajax({
+    url: app.host + '/donuts',
+    method: 'GET',
+    data: data,
```

Comment:
You don't need data in a GET But everything else is correct given that `app.host` is `http://example.com`